tests/cases/fourslash/completionListInImportClause05.ts

```diff
@@ -12,4 +12,4 @@
 // NOTE: The node_modules folder is in "/", rather than ".", because it requires
 // less scaffolding to mock.  In particular, "/" is where we look for type roots.
 
-verify.completionsAt("1", ["@a/b", "@c/d", "@e/f"], { isNewIdentifierLocation: true });
+verify.completionsAt("1", ["@e/f", "@a/b", "@c/d"], { isNewIdentifierLocation: true });

```

Comment:
Shouldn't the results be sorted?

Reply:
They weren't sorted by name before, except by coincidence. We could make a PR sorting them but it would change a bunch of baselines. I think editors do their own sorting on the result anyway.

Reply:
Alright.